Tools/PowershellModule/src/Library/New-ARMObjects.ps1

```diff
@@ -4,29 +4,29 @@ Function New-ARMObjects
 {
     <#
     .SYNOPSIS
-    Creates the Azure Resources to stand-up a Windows Package Manager Rest Source.
+    Creates the Azure Resources to stand-up a Windows Package Manager REST Source.
 
     .DESCRIPTION
     Uses the custom PowerShell object provided by the "New-ARMParameterObject" cmdlet to create Azure resources, and will 
-    create the the Key Vault secrets and publish the Windows Package Manager rest source rest apis to the Azure Function.
+    create the the Key Vault secrets and publish the Windows Package Manager REST source REST apis to the Azure Function.
         
     The following Azure Modules are used by this script:
-        Az.Resources --> Invoke-AzResourceAction
-        Az.Accounts  --> Connect-AzAccount, Get-AzContext
-        Az.Websites  --> Get-AzWebapp
-        Az.Functions --> Get-AzFunctionApp
+        Az.Resources
+        Az.Accounts
+        Az.Websites
+        Az.Functions
 
     .PARAMETER ARMObjects
     Object returned from the "New-ARMParameterObject" providing the paths to the ARM Parameters and Template files.
 
-    .PARAMETER RestSourcePath
-    Path to the compiled Function ZIP containing the Rest APIs
+    .PARAMETER RESTSourcePath
```

Comment:
Don't change scripts on a documentation update. If we are changing script behavior we need an issue/specific PR scoped to that script interface change.

Reply:
fixed